pad_api_data/pad_etl/common/dungeon_parse.py

```diff
@@ -1,12 +1,21 @@
-# A small object to hold floor modifier information
+from dungeon_maps import btypeChart, battrChart
+
+
 class Modifier:
     def __init__(self):
         self.requiredDungeon = 0
-        self.modifiers = []
+        self.remainingModifiers = []
```

Comment:
You should familiarize yourself with the PEP8 style guide:
https://www.python.org/dev/peps/pep-0008/#naming-conventions

In general your code tends not to conform.